sympy/crypto/crypto.py

```diff
@@ -8,6 +8,7 @@
 
 from __future__ import print_function
 
+import random
```

Comment:
why not do `from random import randrange`? instead of this.
I am not sure why one way would be better over the other (will it be?)

Reply:
My bad. My bad. I checked and it seems that there is already a `_randrange` function imported from another module. I used that function instead.